deploy/addon/postgresql/pgpool.md

```diff
@@ -0,0 +1,349 @@
+
+**In Memory Query Cache**: In memory query cache allows to save a pair of SELECT statement and its result. If an identical SELECTs comes in, Pgpool-II returns the value from cache. Since no SQL parsing nor access to PostgreSQL are involved, using in memory cache is extremely fast. On the other hand, it might be slower than the normal path in some cases, because it adds some overhead of storing cache data. 
+
+The different features are available in the [official Pgpool documentation](https://www.pgpool.net/docs/latest/en/html/index.html).
+
+{{< alert "info" "Information:" >}}
+We only support the **Streaming** mode, which is the most used and recommended mode for Pgpool-II. If you need other modes or features that are not supported on Clever Cloud, please contact us at <support@clever-cloud.com> 
+{{< /alert >}}
+
+## How to configure Pgpool-II
+
+### Enable Pgpool-II for your application
+The first thing to do to configure Pgpool-II is to link your PostgreSQL add-on. To do that, you can go to the `Service Dependencies` page of your application
+and select your PostgreSQL add-on.
+
+Once linked, you can enable the Pgpool-II feature by defining the following environment variable: `CC_ENABLE_PGPOOL=true`.
+If you ever need to stop using Pgpool-II, you can remove this variable or set it to `false`.
+
+Your application will use a Unix Domain Socket to connect to Pgpool-II. Unix domain sockets are faster than TCP sockets because there is no handshake and the connection is made locally on the same machine. A special environment variable will be injected to your environment variables: `CC_PGPOOL_SOCKET_PATH`. This variable contains the path to the Unix Domain Socket you have to connect. See [Usages](#usages) below for some example on how to use it.
+
+The available Pgpool-II variables and their descriptions are available on our [environment variables reference page](https://www.clever-cloud.com/doc/reference/reference-environment-variables/).  
+
+### Concurrent session and pool size  
+
+You can use the `CC_PGPOOL_NUM_INIT_CHILDREN` and `CC_PGPOOL_MAX_POOL` environment variables to set the number of **concurrent sessions** and the number of **connection pool caches per connection**.
+
+Let's take an example with 3 PostgreSQL servers, you have configured the `CC_PGPOOL_NUM_INIT_CHILDREN` variable with a value of **32**. 
+
+At startup, Pgpool-II will start **32** processes, one process per connection. The `CC_PGPOOL_MAX_POOL` variable is used to configure the number of cached connections per connection (process). This option is mostly used when you have different credentials for the same PostgreSQL server (database, user...). But in the case of **Streaming** mode on Clever Cloud, you have access to only one database, and the credentials are the same on all add-ons with binary replication. You should not need to change the default value of this variable (**1** by default).
```

Comment:
```suggestion
On startup, Pgpool-II will start **32** processes, one process per connection. The `CC_PGPOOL_MAX_POOL` variable is used to configure the number of cached connections per connection (process). This option is mostly used when you have different credentials for the same PostgreSQL server (database, user...). But in the case of **Streaming** mode on Clever Cloud, you have access to only one database, and the credentials are the same on all add-ons with binary replication. You should not need to change the default value of this variable (**1** by default).
```

Reply:
I have wondered and researched about that one. *On startup* feels more natural to me but it turns out both formulations are [almost interchangeable](https://english.stackexchange.com/questions/154467/at-startup-vs-on-startup).